src/components/organisms/LeafletMap/index.js

```diff
@@ -0,0 +1,112 @@
+import React, { createRef, Component } from 'react'
+import PropTypes from 'prop-types'
+
+import Mapbox from '../MapboxMap'
+import MarkerList from '../../molecules/MarkerList'
+import {
+  isBrowser,
+  GoogleLayer,
+  Map,
+  TileLayer,
+  LayersControl,
+} from '../../../utils/LeafletElements'
+
+const road = 'ROADMAP'
+
+class LeafletWrapper extends Component {
+  state = {
+    isBrowser,
+  }
+
+  componentDidMount() {
+    if (isBrowser) {
+      this.setState({ isBrowser })
+    }
+  }
+
+  setPosition = () => [this.props.lat, this.props.lng]
+
+  mapRef = createRef()
+
+  handleClick = () => {
+    this.mapRef.current.leafletElement.locate()
+  }
+
+  render() {
+    let BaseLayer = null
+    if (isBrowser) {
+      BaseLayer = LayersControl.BaseLayer
+    }
+    const mapDimensions = {
+      width: this.props.width,
+      height: this.props.height,
+    }
+
+    return isBrowser ? (
+      <div>
```

Comment:
Do you really need this `<div>`?

Reply:
Yeap, because of the conditional statement below

Reply:
but, in this case you should use `<Fragment />`